python/sglang/srt/managers/controller/tp_worker.py

```diff
@@ -367,7 +372,33 @@ def get_new_prefill_batch(self) -> Optional[Batch]:
                 ]
             )
 
-        for req in self.forward_queue:
+        # Handle the current inflight request
+        take_inflight = 0
```

Comment:
This function starts to get longer. Consider splitting the body and conditions into multiple subfunctions with good names.

Reply:
Sure, there are still some functions to be implemented, I will organize it in the next PR.